src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+import logoFull from './logo_full.svg';
+import logo from './logo.svg';
+
+/**
+ * The Kids first logo.
+ * May be `full` with `Kids First Data Resource` titling or
+ * `logo mark` with only the art asset.
+ */
+const Logo = ({kind, size, className}) => {
+
+  const logoAsset = kind === 'logo mark' ? logo : logoFull;
+  const logoClasses = classes(
+      className,
+      'Logo',
+      {[`Logo--${size}`] : size !== 'medium'},
+  )
+  const alt = 'Kids First Data Resource Center logo';
+
+  return (
+    <div className={logoClasses}>
+      <img
+        className={logoClasses}
+        alt={alt}
+        src={logoAsset}
+      />
+      <h1 className='screen-reader-text'>{alt}</h1>
```

Comment:
rename to `text-screen-reader` to stay consistent with the syntax of other text util classes